gpfit/tests/t_examples.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+import numpy as np
+import sys
+import os
+import importlib
+
+
+FILE_DIR = os.path.dirname(os.path.realpath(__file__))
+EXAMPLE_DIR = os.path.abspath(FILE_DIR+'../../../docs/source/examples')
+IMPORTED_EXAMPLES = {}
+
+
+class TestExamples(unittest.TestCase):
+
+    # To test a new example, add a function called `test_$EXAMPLENAME`, where
+    # $EXAMPLENAME is the name of your example in docs/source/examples without
+    # the file extension.
+    #
+    # This function should accept two arguments (e.g. 'self' and 'example').
+    # The imported example script will be passed to the second: anything that
+    # was a global variable (e.g, "sol") in the original script is available
+    # as an attribute (e.g., "example.sol")
+    #
+    # If you don't want to perform any checks on the example besides making
+    # sure it runs, just put "pass" as the function's body, e.g.:
+    #
+    #       def test_dummy_example(self, example):
+    #           pass
+    #
+    # But it's good practice to ensure the example's solution as well, e.g.:
+    #
+    #       def test_dummy_example(self, example):
+    #           self.assertAlmostEqual(example.sol["cost"], 3.121)
+    #
+
+    def test_t_ex6_1(self, example):
+        pass
+
```

Comment:
@bqpd: thoughts on importing `StdoutCaptured` and `new_test` from gpkit instead of re-implementing them here? (obviously, we'd have to add some kwargs so that the constant dirnames can get passed correctly)